src/Event/BasicAuth/BasicAuthInterface.php

```diff
@@ -72,7 +72,7 @@ public function setMatcher($matcher);
     /**
      * Authenticates a request.
      *
-     * @param \Ivory\HttpAdapter\Event\BasicAuth\InternalRequestInterface $internalRequest The internal request.
+     * @param \Ivory\HttpAdapter\Message\InternalRequestInterface $internalRequest The internal request.
```

Comment:
I'm currently thinking to switch everything to short class name and add missing use statements everywhere. So, you can let you as it for now.